packages/terra-list/package.json

```diff
@@ -42,7 +42,8 @@
     "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
     "lint:scss": "stylelint src/**/*.scss",
     "test": "npm run test:jest && npm run test:nightwatch",
-    "test:jest": "jest --config ../../jestconfig.js",
-    "test:nightwatch": "nightwatch -c ../../nightwatch.conf.js"
+    "test:jest": "jest ./tests/jest/* --config ../../jestconfig.json",
+    "test:nightwatch": "nightwatch -c ../../nightwatch.conf.js",
+    "test:wdio": "../../node_modules/.bin/wdio ../../node_modules/terra-dev-site/config/wdio/wdio.conf.js"
```

Comment:
Define terra-base as peer dependency only. 
https://github.com/cerner/terra-core/issues/1587

Reply:
Removed the dependency - [Github](https://github.com/cerner/terra-core/pull/1596/files#diff-35a5fbde00aa67f33c9cc6c56919273a)